xcdat/axis.py

```diff
@@ -164,9 +164,9 @@ def _gen_bounds(self, axis: Axis, width: float = 1) -> xr.DataArray:
         self._dataset[var_name] = xr.DataArray(
             name=var_name,
             data=bounds,
-            coords={axis: axis_coords.data},
+            coords={axis: axis_coords},
```

Comment:
This was the root cause for why attributes were being dropped from the axis coords. 

Basically, the bounds var was setting its axis coords to just the `np.ndarray` of the axis coords, not the entire DataArray which includes the attributes. As a result, it also replaced the existing axis coords.